README.md

```diff
@@ -29,17 +29,20 @@ We are always looking for help. Working on this project is an opportunity to use
 
     - Go to https://console.firebase.google.com/ and click "Add project";
     - Enter "Boxwise Development" into the name field and click "Create Project";
-    - Once that has completed, click "Add Firebase to your web app". Ignore the prompt to copy and paste the entire snippet, and look at the fields in the `config` object;
-    - Create a new local config running `yarn setup`, filling with the Firebase project ID (you may be asked to login). This will create a `.env.local` file;
+    - Once that has completed, click "Add Firebase to your web app". Ignore the prompt to copy and paste the snippet for now;
```

Comment:
This step can be removed entirely - all it does is open the modal with the config details, I think.